apps/console/src/features/applications/components/settings/attribute-management/attribute-selection-oidc.tsx

```diff
@@ -132,23 +144,27 @@ export const AttributeSelectionOIDC: FunctionComponent<AttributeSelectionOIDCPro
 
     const [ initializationFinished, setInitializationFinished ] = useState<boolean>(false);
 
-    const initValue = useRef(false);
+    const initValue: MutableRefObject<boolean> = useRef<boolean>(false);
 
     const config: ConfigReducerStateInterface = useSelector((state: AppState) => state.config);
 
     const [ openIDConnectClaims ] = useState<ExtendedExternalClaimInterface[]>(externalClaims);
 
+    const [ selectedScopes, setSelectedScopes ] = useState([ 
+        t("console:develop.features.applications.edit.sections.attributes.selection.constants.openid")
```

Comment:
We don't need to localize a constant. You can just create a constant in the file and then use it. For instance:
```
const OPENID = "opened";
```
And then you can use `OPENID` wherever you want.

Reply:
resolved with https://github.com/wso2/identity-apps/pull/3640/commits/36d8f442d1333ae449424157d70d879c508e5d43